Makefile

```diff
@@ -0,0 +1,90 @@
+.DEFAULT_GOAL		:=help
+SHELL				:=/bin/bash
+
+.PHONY: help
+help: ## Show this help
+	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: dev
+dev:
+	$(eval DEPLOY_ENV=dev)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+
+read-keyvault-config:
+	$(eval KEY_VAULT_NAME=$(shell jq -r '.key_vault_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+	$(eval KEY_VAULT_SECRET_NAME=$(shell jq -r '.key_vault_secret_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
```

Comment:
key_vault_secret_name is not provided in paramaeters file

Reply:
I think we could hardcode the secret name here since it's same for all environments.  Also putting it in t tfvars.json doesn't make sense as it's not related to terraform.

`$(eval KEY_VAULT_SECRET_NAME=APPLY-QTS-APP-VARIABLES)`  

@merrilmathew

Reply:
I think ok to hardcode for now since there is only one secret in KV for QTS